app/src/main/java/us/shandian/giga/ui/fragment/MissionsFragment.java

```diff
@@ -208,6 +208,14 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
     }
 
+    public void showDeleteFileConfirmationAlert(){
+        AlertDialog.Builder prompt = new AlertDialog.Builder(mContext);
+        prompt.setTitle(R.string.confirm_delete_all_downloads);
+        prompt.setPositiveButton(R.string.no,  null);
+        prompt.setNegativeButton(R.string.yes, (dialog, which) -> mAdapter.clearFinishedDownloads(true));
```

Comment:
@darnuria The actions of positive/negative buttons are still inverted. Normally the positive button exectues the action and has some positive string (like "yes") and vice versa.

I already mentioned it in my initial review, but should have not grouped it together with the another point, since it seems to have been lost somewhere in there.